wcivf/apps/elections/models.py

```diff
@@ -49,9 +49,37 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @property
     def in_past(self):
+        """
+        Returns a boolean for whether the election date is in the past
+        """
         return self.election_date < datetime.date.today()
 
+    @property
+    def is_city_of_london(self):
+        """
+        Returns boolean for if the election is within City of London district
```

Comment:
Can you expand this comment a little to explain _why_ we special case the city? Just "The city often has different rules to other UK elections so it's useful to know when we need to special case" or something along those lines.